src/main/java/com/twilio/oai/TwilioJavaGenerator.java

```diff
@@ -837,6 +867,17 @@ private CodegenParameter resolveEnumParameter(CodegenParameter parameter, String
         return parameter;
     }
 
+    private String getSubDomainName(Map<String, String> subDomainMap, String name) {
+        String subDomainName = null;
+        for (Map.Entry<String, String> set :
+                subDomainMap.entrySet()) {
+            if(set.getKey().equalsIgnoreCase(name)){
```

Comment:
I would suggest have subDomainMap key to be in specific format(UpperCase/Lowercase). I see we are updating it here https://github.com/twilio/twilio-oai-generator/pull/201/files#diff-f43623484339759ddb9778b52b89a94fecb327bf721adc186e33db0067ece174R400. That way you would not need to iterate over the map, instead fetch it directly.

Reply:
subDomainMap key is later used to name the folders within the domain. This name is case sensitive and cannot be modified to either be in. uppercase or lowercase completely. Modified the getSubDomain function to use stream as suggested
Ticket to track the efforts for implement the current code changes to be used for all domains without versions : https://issues.corp.twilio.com/browse/DII-582